VisionProSurgery/WebView.swift

```diff
@@ -0,0 +1,51 @@
+//
+// This source file is part of the StanfordBDHG VisionProSurgery project
+//
+// SPDX-FileCopyrightText: 2024 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+import WebKit
+
```

Comment:
Generally want to add two empty lines between the import and types. Also applies to elements between content elements in types as suggested in this file.
```suggestion


```